### PR TITLE
message error fixed

### PR DESCRIPTION
**Pull request by daisyzarzo**
realtes to https://github.com/bespoken/source-api/issues/128

### CODE REVIEW

File: src/main/firebase/FirebaseController.ts [hunk 1]
```diff
@@ -137,20 +136,17 @@ export class FirebaseDatabase {
         } catch (error) {
             success = false;
             errorMessage = error.message ? error.message : error.toString();
+            console.error(errorMessage);
```
Comment (ecruzado) [inline src/main/firebase/FirebaseController.ts]: you probably need more context to try to find it. like  console.error("errorOnFunction", errorMessage);

File: src/main/services/v1/Users.ts [hunk 2]
```diff
@@ -69,7 +69,8 @@ export function deleteUserAccount(adminAuth: Admin.auth.Auth, db: Admin.database
                 Returns.Okay(res).send();
                 return res;
             } else {
-                Returns.NotOkay(res, 400, "there is no userId present or valid").send();
+                console.error("There is no userId present or valid");
```
Comment (ecruzado) [inline src/main/services/v1/Users.ts]: same here, probably need more context in the logging like the user you are validating

File: src/main/firebase/FirebaseController.ts (hunk 1 above)
Comment (daisyzarzo) [reply src/main/firebase/FirebaseController.ts]: here is problems trying connect with stripe-api to delete customer Error: "value" required in setHeader("x-access-token", value)

File: src/main/services/v1/Users.ts (hunk 2 above)
Comment (daisyzarzo) [reply src/main/services/v1/Users.ts]: this message shows itself only when the res.body, not contain userID

**Pull request merged**